### PR TITLE
Added the bundles for infix-en_GB

### PR DESCRIPTION
**Pull request by yiss**
----
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/master/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

File: apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/AnyAssertionsSpec.kt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package ch.tutteli.atrium.api.infix.en_GB
 
 import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.creating.AnyAssertionsBuilder.toBe
```
Comment (robstoll) [inline apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/AnyAssertionsSpec.kt]: please revert all changes in this file, this is reason why the build fails

File: bundles/infix-en_GB-jdk8/atrium-infix-en_GB-jdk8-jvm/src/test/kotlin/custom/SmokeSpec.kt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        // TODO #233 uncomment once toBe is implemented
+        //assertThat(1) toBe 2
+    }
+
+    test("see if `Path.existsNot` can be used") {
+        assertThat(Paths.get("nonExisting"))
+    }
+
+    test("see if own assertion function without i18n can be used") {
+        // TODO #233 uncomment once toBe is implemented
```
Comment (robstoll) [inline bundles/infix-en_GB-jdk8/atrium-infix-en_GB-jdk8-jvm/src/test/kotlin/custom/SmokeSpec.kt]: you can remove the todo once you implement `toBe even` below.

File: bundles/infix-en_GB-jdk8/atrium-infix-en_GB-jdk8-jvm/src/test/kotlin/custom/SmokeSpec.kt [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        //assertThat(2) toBe even
+    }
+
+    test("see if own assertion function with i18n can be used") {
+        assertThat(4).isMultipleOf(2)
+    }
+})
+
+fun Expect<Int>.isEven(): Expect<Int> =
```
Comment (robstoll) [inline bundles/infix-en_GB-jdk8/atrium-infix-en_GB-jdk8-jvm/src/test/kotlin/custom/SmokeSpec.kt]: Please implement it as infix function, see SmokeSpec in atrium-cc-infix-en_GB-robstoll for an example

File: bundles/infix-en_GB-jdk8/atrium-infix-en_GB-jdk8-jvm/src/test/kotlin/custom/SmokeSpec.kt [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    createAndAddAssertion(DescriptionBasic.IS, RawString.create("an even number")) { it % 2 == 0 }
+
+fun Expect<Int>.isMultipleOf(base: Int): Expect<Int> = addAssertion(_isMultipleOf(this, base))
```
Comment (robstoll) [inline bundles/infix-en_GB-jdk8/atrium-infix-en_GB-jdk8-jvm/src/test/kotlin/custom/SmokeSpec.kt]: same same, please as infix function

File: bundles/infix-en_GB-kotlin_1_3/atrium-infix-en_GB-kotlin_1_3-android/build.gradle [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    api prefixedProject('domain-api-android')
+    api prefixedProject('core-api-android')
+
+    runtimeOnly prefixedProject('domain-robstoll-android')
+    runtimeOnly prefixedProject('core-robstoll-android')
+}
+
+srcAndResourcesFromJvmProject(project)
+
+compileKotlin.kotlinOptions.languageVersion = '1.3'
```
Comment (robstoll) [inline bundles/infix-en_GB-kotlin_1_3/atrium-infix-en_GB-kotlin_1_3-android/build.gradle]: this line shouldn't be necessary. If you have copied it from atrium-fluent-en_GB-kotlin_1_3-android, then please remove it there as well

File: bundles/infix-en_GB-kotlin_1_3/atrium-infix-en_GB-kotlin_1_3-common/build.gradle [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+dependencies {
+    api prefixedProject('verbs-common')
+    api prefixedProject('api-infix-en_GB-kotlin_1_3-common')
+    api prefixedProject('translations-en_GB-common')
+    api prefixedProject('domain-builders-common')
+    api prefixedProject('domain-api-common')
+    api prefixedProject('core-api-common')
+}
+
+compileKotlinCommon.kotlinOptions.languageVersion = '1.3'
```
Comment (robstoll) [inline bundles/infix-en_GB-kotlin_1_3/atrium-infix-en_GB-kotlin_1_3-common/build.gradle]: same same as previous comment, please remove this line (and in fluent)

File: bundles/infix-en_GB-kotlin_1_3/atrium-infix-en_GB-kotlin_1_3-js/build.gradle [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    api prefixedProject('translations-en_GB-js')
+    api prefixedProject('domain-builders-js')
+    api prefixedProject('domain-api-js')
+    api prefixedProject('core-api-js')
+
+    implementation prefixedProject('domain-robstoll-js')
+    implementation prefixedProject('core-robstoll-js')
+}
+
+compileKotlin2Js.kotlinOptions.languageVersion = '1.3'
```
Comment (robstoll) [inline bundles/infix-en_GB-kotlin_1_3/atrium-infix-en_GB-kotlin_1_3-js/build.gradle]: same same as previous comment, please remove this line (and in fluent)

File: bundles/infix-en_GB-kotlin_1_3/atrium-infix-en_GB-kotlin_1_3-js/src/main/kotlin/ch/tutteli/atrium/infix/en_GB/kotlin_1_3/dependOnMe.kt [hunk 8]
```diff
@@ -0,0 +1,15 @@
+package ch.tutteli.atrium.infix.en_GB.kotlin_1_3
+
+import ch.tutteli.atrium.core.robstoll.dependOn_atrium_core_robstoll
+import ch.tutteli.atrium.domain.robstoll.dependOn_atrium_domain_robstoll
+
+/**
+ * Dummy function in order that other modules can define a dependency on atrium-fluent-en_GB-kotlin_1_3-js
```
Comment (robstoll) [inline bundles/infix-en_GB-kotlin_1_3/atrium-infix-en_GB-kotlin_1_3-js/src/main/kotlin/ch/tutteli/atrium/infix/en_GB/kotlin_1_3/dependOnMe.kt]:
```suggestion
 * Dummy function in order that other modules can define a dependency on atrium-infix-en_GB-kotlin_1_3-js
```

File: bundles/infix-en_GB-kotlin_1_3/atrium-infix-en_GB-kotlin_1_3-jvm/build.gradle [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+    api prefixedProject('translations-en_GB-jvm')
+    api prefixedProject('domain-builders-jvm')
+    api prefixedProject('domain-api-jvm')
+    api prefixedProject('core-api-jvm')
+
+    runtimeOnly prefixedProject('domain-robstoll-jvm')
+    runtimeOnly prefixedProject('core-robstoll-jvm')
+}
+
+compileKotlin.kotlinOptions.languageVersion = '1.3'
```
Comment (robstoll) [inline bundles/infix-en_GB-kotlin_1_3/atrium-infix-en_GB-kotlin_1_3-jvm/build.gradle]: same same as previous comment, please remove this line (and in fluent)

File: bundles/infix-en_GB-kotlin_1_3/atrium-infix-en_GB-kotlin_1_3-jvm/src/test/kotlin/custom/SmokeSpec.kt [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+    test("see if own assertion function with i18n can be used") {
+        //TODO #233 uncomment once toBe is implemented
+        //assertThat(4) toBe multipleOf 2
+    }
+})
+
+fun Expect<Int>.isEven(): Expect<Int> =
+    createAndAddAssertion(DescriptionBasic.IS, RawString.create("an even number")) { it % 2 == 0 }
+
+fun Expect<Int>.isMultipleOf(base: Int): Expect<Int> = addAssertion(_isMultipleOf(this, base))
```
Comment (robstoll) [inline bundles/infix-en_GB-kotlin_1_3/atrium-infix-en_GB-kotlin_1_3-jvm/src/test/kotlin/custom/SmokeSpec.kt]: same same, implement as infix

File: bundles/infix-en_GB/atrium-infix-en_GB-common/src/test/kotlin/SmokeTest.kt [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+//        assertThat {
+//
+//        }.notToThrow()
+    }
+}
+
+
+fun Expect<Int>.isEven() = createAndAddAssertion(IS, RawString.create("an even number")) { it % 2 == 0 }
+
+fun Expect<Int>.isMultipleOf(base: Int): Expect<Int> = addAssertion(_isMultipleOf(this, base))
```
Comment (robstoll) [inline bundles/infix-en_GB/atrium-infix-en_GB-common/src/test/kotlin/SmokeTest.kt]: same same, implement as infix

File: bundles/infix-en_GB/atrium-infix-en_GB-js/src/main/kotlin/ch/tutteli/atrium/infix/en_GB/dependOnMe.kt [hunk 12]
```diff
@@ -0,0 +1,15 @@
+package ch.tutteli.atrium.infix.en_GB
+
+import ch.tutteli.atrium.core.robstoll.dependOn_atrium_core_robstoll
+import ch.tutteli.atrium.domain.robstoll.dependOn_atrium_domain_robstoll
+
+/**
+ * Dummy function in order that other modules can define a dependency on atrium-fluent-en_GB-js
```
Comment (robstoll) [inline bundles/infix-en_GB/atrium-infix-en_GB-js/src/main/kotlin/ch/tutteli/atrium/infix/en_GB/dependOnMe.kt]:
```suggestion
 * Dummy function in order that other modules can define a dependency on atrium-infix_GB-js
```

File: gradle/wrapper/gradle-wrapper.properties [hunk 13]
```diff
@@ -1,5 +1,6 @@
+#Sun Oct 20 17:29:30 CEST 2019
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.5.1-all.zip
```
Comment (robstoll) [inline gradle/wrapper/gradle-wrapper.properties]: please revert the changes in this file

**Comment by yiss** [comment]
I'll try to make the changes you suggested tonight

**Comment by robstoll** [comment]
@yiss let me know in case you are stuck and need help

**Comment by yiss** [comment]
@robstoll thank you. I just wasn't fimiliar with the infix functions in Kotlin so I had to see how it works. I'll try to revise my PR this weekend

**Comment by robstoll** [comment]
@yiss :+1: and no stress, just wanted to be sure you are not stuck

**Comment by robstoll** [comment]
@yiss let me know in case you need help with the infix behaviour in Kotlin or in case you have troubles rebasing your branch on master (it is out of date and has conflicts)

**Comment by yiss** [comment]
@robstoll Hi sorry I totally forgot about this I'll try to finish it this week, thanks for the reminder.

**Comment by robstoll** [comment]
@yiss take a look at ch.tutteli.atrium.api.infix.en_GB.CollectionAssertionsSpec, this should help you with your AnyAssertionsSpec I guess.

**Comment by robstoll** [comment]
@yiss I bet your New Year's resolution is to finish this PR :wink: 
Let me know in case you need help

**Comment by robstoll** [comment]
Closing it as there is no activity, re-open it in case you want to work on it again
